crates/bevy_reflect/src/type_registry.rs

```diff
@@ -599,6 +599,23 @@ impl<T: Reflect> FromType<T> for ReflectFromPtr {
     }
 }
 
+/// In order to dynamic get component's dyn object.
+///
+/// This trait associates the dyn object corresponding to [`TypeData`] (because rust does not currently support `dyn T`. `T` is a generic parameter.),
```

Comment:
```suggestion
/// This trait associates the dyn object corresponding to [`TypeData`] (because Rust does not currently support `dyn T`, where `T` is a generic parameter).
```